### DISCUSSION TITLE
What is the difference between this and VTM?

### DISCUSSION THREAD

**Discussion by stephendliang** [fraunhoferhhi/vvenc#216] [General]
I am honestly wondering what the difference between VVenC and VTM is. I apologize, I am very confused.

My current impression is that this is a faster encoder for regular use, while VTM is a somewhat more accurate reference encoder. Is that so?

**Comment by adamjw24** [comment]
Your impression is totally correct.

Most of all, those are two different encoders, developed by different people (well, mostly), for other reasons and tasks. The only thing the two projects have in common: VVenC was started from the VTM codebase like 3.5 years ago, but it diverged strongly since then.
